vmslave-config/ansible/roles/sync-clock/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+- name: Install ntp service
+  become: true
+  apt: name={{item}} state=present
+  with_items:
+    - ntp
+
+- name: stop ntp service
+  become: true
+  service:
+    name: ntp
+    state: stopped
+
+- name: config ntp service
+  become: true
+  template:
+    src: ntp.conf
```

Comment:
is it possible that you echo 2 new lines into /etc/ntp.conf, instead of maintain an own ntp.conf which most part are not our concern ?